Automattic/wp-calypso
# HelpCenter: replace broken icons

## Changes proposed in this Pull Request

Some icons are not showing in HelpCenter due to an issue with the CDN URL. This PR replaces the `chevron-left` icons with the `wordpress/icon` version and the `MaterialIcon` new-releases with a static version in the `inline-help` component.

## Testing instructions

1. Pull branch and `cd apps/editing-toolkit && yarn dev --sync`
2. Check that the icons are loading as proper SVG's not using `<use />` as this was part of the problem.
3. Check the What's New icon on the first section and the Back icon in Live Chat/ Email contact forms.

Fixes #63491
